server-main.go

```diff
@@ -135,10 +135,10 @@ func initServerConfig(c *cli.Context) {
 	// Validate if both keys are specified and they are valid save them.
 	if accessKey != "" && secretKey != "" {
 		if !isValidAccessKey.MatchString(accessKey) {
-			fatalIf(errInvalidArgument, "Access key does not have required length", nil)
+			fatalIf(errInvalidArgument, "Access key does not have required length")
```

Comment:
Invalid access key.

isValidAccessKey() -> could fail for other reasons as well in the future.